src/app/static/src/tests/genericChart/mutations.test.ts

```diff
@@ -13,13 +12,9 @@ describe("genericChart mutations", () => {
 
     it("sets dataset",  () => {
         const state = mockGenericChartState();
-        const vueSetSpy = jest.spyOn(Vue, "set");
```

Comment:
The line which was calling Vue.set has been removed in one of the other vue3 migration PRs but I think we don't need to test this anymore? But not 100% sure. Sounds like this call isn't required see e.g. https://stackoverflow.com/questions/69780482/how-do-i-do-vue-set-in-vue-3

Reply:
Yeah, looks like `Vue.set` isn't called any more, but the `state.selectionDatasetId` value is also set in the mutation, so should test that too here.

Reply:
Thanks, added a test for that